vault/core.go

```diff
@@ -477,14 +477,31 @@ func (c *Core) handleRequest(req *logical.Request) (retResp *logical.Response, r
 	// If there is a secret, we must register it with the expiration manager.
 	// We exclude renewal of a lease, since it does not need to be re-registered
 	if resp != nil && resp.Secret != nil && !strings.HasPrefix(req.Path, "sys/renew/") {
+		// Get the SystemView for the mount
+		sysView, err := c.sysViewByPath(req.Path)
+		if err != nil {
```

Comment:
The log statements are already in that format, so they're just written verbatim here to avoid [ERR] core: [ERR] core: ...

As for whether the path could ever result in an error, the lookup functions that the path is being sent to return errors, so I deferred to that (for the reasons above).